react/BottomSheet/BottomSheet.jsx

```diff
@@ -279,6 +278,9 @@ const BottomSheet = memo(
           </div>
           <div style={{ height: backdrop ? 0 : bottomSpacerHeight }} />
         </MuiBottomSheet>
+        {getFlagshipMetadata().immersive && (
+          <span style={styles.flagshipImmersive} />
+        )}
```

Comment:
OK so it works now for the bottom, but let's populate the bottomsheet with a lot of content in order to have the top of the bottom sheet above your screen. What happens? Don't we have to also put this span at the top of the bottomsheet?

Reply:
In fact, I don't understand the need of this `<span` since now we have a padding ?

Reply:
I did not succeed in adding a margin bottom to the BottomSheet (maybe not possible ?). So I think that is the purpose of the `span` : taking the space of the Android buttons.

But for me it did not work as is, so I add to move it down (why ?).

But then it hides the last item of the BottomSheet. That's why I added the paddingBottom.

Reply:
This looks safe to me, on one condition: the bottomSheet in `immersive` mode is never child of something giving it the Native bottom padding (which should not be the case as it is a fixed element)

Reply:
@zatteo padding-bottom is not sufficient by itself?

Reply:
Nope just adding a padding-bottom is not enough :/